### ISSUE TITLE
prevent auto from updating its observers

### ISSUE THREAD

**Issue by ryanking1809** [alloc/wana#34]
Hey, sorry to keep finding the most obscure use cases.
It looks like `auto` updates its observers on each run. When it comes to making these deferred updates, it forces me to do some unnecessary computation, and I wonder if there's any easy way around it.

```js
useEffect(() => {
    const a = auto(() => {
      // would be nice to fire only once but necessary to maintain observation
      // might be a complex getter which would defeat the purpose of deferring the update
      readObservableValue();
      // defer update until browser is idle
      requestIdleCallback(() => updateReactState());
    });
    return () => {
      console.log("disposed");
      a.dispose();
    };
  }, []);
  ```

Is there a way to set the auto function observables in the initial run and then never refer to them again them again?
A simplified example would be this.
```js
const store = o({ test: 1 });
auto(function () {
  // this just exists to set up the initial observation
  // and should never be used again
  if (!this.fired) {
    const val = store.test; // read observable object
    console.log("fire once");
    this.fired = true;
  }
  // this is what I would like to happen every time
  console.log("fire multiple");
});
store.test++; // fire once, fire multiple
store.test++; // fire multiple
store.test++; // stops observing -> would like "fire multiple" again
store.test++; // stops observing -> would like "fire multiple" again
store.test++; // stops observing -> would like "fire multiple" again
```

**Comment by ryanking1809** [comment]
Awesome, thanks! but it seems to not be working on my end. I tried playing around a bit, but I don't know enough about the internals to get it operational.

I've used my example above at the top of `App.js`, and there's an `observeOnce.js` file for your code. It seems to fire once but never again.
https://codesandbox.io/s/wana-deferred-update-b7f0t?file=/src/App.js
